man/systemd.network.xml

```diff
@@ -842,6 +842,15 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>Preference=</varname></term>
```

Comment:
If this is specific to IPv6, I'd prefer if we'd prefix this with "IPv6", so that this becomes `IPv6Preference=`...